processor/src/main/kotlin/permissions/dispatcher/processor/exception/AmbiguousMethodException.kt

```diff
@@ -0,0 +1,6 @@
+package permissions.dispatcher.processor.exception
+
+import permissions.dispatcher.processor.util.simpleString
+import javax.lang.model.element.ExecutableElement
+
+class AmbiguousMethodException(e: ExecutableElement) : RuntimeException("'${e.simpleString()}()' has duplicated '@NeedsPermission' method. The method annotated with '@NeedsPermission' must has the unique name.")
```

Comment:
How about `DuplicatedMethodNameException`? From my perspective it's a bit intuitive.